ion-schema/src/main/kotlin/com/amazon/ionschema/reader/IonSchemaReader.kt

```diff
@@ -21,7 +21,7 @@ interface IonSchemaReader {
      * the given Ion. Reporting all errors is a best-effort attempt because the presence of one error can mask other
      * errors.
      */
-    fun readSchema(document: List<IonValue>, failFast: Boolean = false): IonSchemaResult<SchemaDocument, List<ReadError>>
+    fun readSchema(document: Iterable<IonValue>, failFast: Boolean = false): IonSchemaResult<Pair<SchemaDocument, Iterator<IonValue>>, List<ReadError>>
```

Comment:
(Suggestion) Add a comment for what the `Pair<SchemaDocument, Iterator<IonValue>>` indicates here

Reply:
No longer applicable because I'm rolling back the change to the return type.